### PR TITLE
Release 6.0.3

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Please write a sentence or two summarizing the proposed change.

Addresses [ORCA-643: Investigate and fix API Gateway authentication errors](https://bugs.earthdata.nasa.gov/browse/ORCA-643)

## Changes

* Removed unused security policy
* Reverted IAM authorization

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Deployed to AWS.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] CHANGELOG.md
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: modules/api-gateway/main.tf [hunk 1]
```diff
@@ -21,39 +21,6 @@ resource "aws_api_gateway_rest_api" "orca_api" {
...
-        "StringNotEquals": {
-          "aws:SourceVpc": "${var.vpc_id}"
-        }
-      }
-    }
-  ]
-}
-EOF
-}
-
```
Comment (rizbihassan) [inline modules/api-gateway/main.tf:56]: these changes were not added in orca-437. Do we still want to remove it?
This policy denies API invoke calls made outside vpc.

**Review by rizbihassan — COMMENTED** [review]
one comment on removing the resource policy.

File: modules/api-gateway/main.tf (hunk 1 above)
Comment (AndrewDorn) [reply modules/api-gateway/main.tf:56]: It would, if NGAP didn't remove it. Best I can tell, this was never left active for more than a few milliseconds.

Comment (rizbihassan) [reply modules/api-gateway/main.tf:56]: I checked my api resource policy and that looks way different than what I added. So NGAP changed the policy in the backend without informing us.

Comment (rizbihassan) [reply modules/api-gateway/main.tf:56]: Looks like communication between teams is becoming a problem and this is not the first time we are facing the consequences :)

Comment (rizbihassan) [reply modules/api-gateway/main.tf:56]: FYi @Quarriers89 @rykearney

**Review by rizbihassan — APPROVED** [review]

**Pull request merged**